portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,33 @@
     <meta charset="UTF-8">
     <title>3030</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <script src="script.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js" integrity="sha384-1CmrxMRARb6aLqgBO7yyAxTOQE2AKb9GfXnEo760AUcUmFx3ibVJJAzGytlQcNXd" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/96f9ad965c.js" crossorigin="anonymous"></script>
   </head>
   <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">Portfolio</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+        <div class="navbar-nav">
+          <a class="nav-item nav-link active" href="#">About me <span class="sr-only">(current)</span></a>
+          <a class="nav-item nav-link" href="#">Projects</a>
+          <a class="nav-item nav-link" href="#">Gallery</a>
+          <a class="nav-item nav-link" href="#">Fun</a>
+        </div>
+      </div>
+    </nav>
     <div id="content">
       <h1>Mariano Shaar</h1>
       <a href="https://github.com/shaargtz" class="link">
-        <img src="./images/icons/github.svg" alt="GitHub link icon" class="icon">
+        <i class="fab fa-github"></i>
```

Comment:
Can you still include the alt field for these icons as well? In general we want to include the alt fallback tag for all multimedia (images, videos, animations, icons). If the fontawesome icons already have the alt tag then that's fine then! (And same goes for the linkedIn icon below)

Reply:
Done!